forge-gui/res/cardsfolder/upcoming/errant_street_artist.txt

```diff
@@ -0,0 +1,9 @@
+Name:Errant, Street Artist
+ManaCost:U
+Types:Legendary Creature Human Rogue
+PT:0/3
+K:Flash
+K:Defender
+K:Haste
+A:AB$ CopySpellAbility | Cost$ 1 U T | ValidTgts$ Spell.YouCtrl+wasNotCast | TargetType$ Spell | MayChooseTarget$ True | AILogic$ OnceIfViable | SpellDescription$ Copy target spell you control that wasn’t cast. You may choose new targets for the copy.

```

Comment:
`’` -> `'`

I won't put this comment on any other cards, but please check the rest too.
Forge doesn't handle "curly" apostrophes and quotation marks well